test/registry.jl

```diff
@@ -145,15 +145,6 @@ end
         @test is_pkg_available(Example1)
         @test is_pkg_available(Example2)
 
-        # Behaviour with conflicting registry names
-        @test_throws PkgError pkgstr("registry up RegistryFoo")
```

Comment:
@fredrikekre I think these tests are redundant with this since you can't have to registries with the same name in the same depot, and `registry rm` now only looks at the first depot. Do you agree?